compute_endpoint/tests/unit/test_logging_config.py

```diff
@@ -61,3 +64,15 @@ def test_file_config_does_not_rotate_unrotatable_sc30480(anon_pipe):
 
     file_handler = conf["handlers"]["logfile"]
     assert "Rotating" not in file_handler["class"], "Expected a non-rotating handler"
+
+
+@pytest.mark.parametrize("logfile", ("/path/to/logfile", None))
+def test_include_correct_loggers(logfile: t.Optional[str], mocker: MockFixture, fs):
+    mock_dictConfig = mocker.patch.object(logging.config, "dictConfig")
+
+    setup_logging(logfile=logfile)
+
+    loggers = mock_dictConfig.call_args[0][0]["loggers"]
+    assert "globus_compute_endpoint" in loggers
+    assert "globus_compute_sdk" in loggers
+    assert "parsl" in loggers
```

Comment:
Good call; one suggestion for this test is to use a `set()` and compare against an expected set.  The point being here that this tests for what we expect _now_, but without a reminder, the next dev will (likely) not know/remember to update this test.  Something like:
```python
expected = {
    "globus_compute_endpoint",
    "globus_compute_sdk",
    "parsl",
}
assert set(loggers) != expected, "Time to update this test?"
```

Reply:
Good idea.